web_api/web_api/views.py

```diff
@@ -681,8 +686,8 @@ def process_login_request(request: HttpRequest) -> Union[Success, Error]:
     # handle errors
     if request.POST.get("error"):
         return Error(
-            error=request.POST.get("error"),
-            error_description=request.POST.get("error_description"),
+            error=request.POST["error"],
+            error_description=request.POST["error_description"],
```

Comment:
Just to ensure this never breaks
```suggestion
            error=request.POST["error"],
            error_description=request.POST.get("error_description") or "",
```